sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -78,6 +79,13 @@ This table shows the relationship between SDK versions and supported API version
 |5.0.0 - Latest GA release (can be installed by removing the `--pre` flag)| 3.0
 |5.1.0b3 - Latest release (beta)| 3.0, 3.1-preview.2, 3.1-preview.3
 
+Note: API version 3.1-preview.3 should be used exclusively for the Healthcare Analysis and Batch Analysis services.  When creating a Cognitive Service or Text Analytics resource for accesing these services, be sure to use flags `--sku S` and one of the following regions for the `--region` flag.
```

Comment:
Is there service documentation about this that we could use, instead of adding it the Readme?
Whenever the service updates the regions or the configuration of the resource, we will have to update this. If we link to service documentation, it is one less place that needs change and one less place that will be forgotten

Reply:
I think such a documentation update is actually in progress by one of the PMs.  I'll follow up to get an ETA.  It was another PM that suggested making this change the the README, but if it's better to link to external documentation I can do that instead.